aea/context/base.py

```diff
@@ -29,6 +29,8 @@
 from aea.mail.base import Address, OutBox
 from aea.skills.tasks import TaskManager
 
+DEFAULT_OEF = "default_oef"
```

Comment:
Note, @marcofavorito we need to expose the choice of OEF address on the configuration. For now it is stubbed.